File: Euclid/Messages/Incoming/GETSTRIP.cs
using Euclid.Game;
using Euclid.Messages.Outgoing;
using Euclid.Network.Streams;
using Euclid.Util.Extensions;

namespace Euclid.Messages.Incoming
{
    class GETSTRIP : IMessageEvent
    {
        public void Handle(Player player, Request request)
        {
            player.Inventory.TurnPage(request.Content);
        }
    }
}

File: euclid/src/main/java/euclid/messages/incoming/GETSTRIP.java
package euclid.messages.incoming;

import euclid.game.Player;
import euclid.messages.IMessageEvent;
import euclid.network.streams.Request;

public class GETSTRIP implements IMessageEvent {

    @Override
    public void handle(Player player, Request request) {
        player.getInventory().turnPage(request.getContent());
    }
}
